guides/source/command_line.md

```diff
@@ -533,30 +885,290 @@ Ruby version              3.2.0 (x86_64-linux)
+bin/rails db:version             # Retrieve the current schema version number
+bin/rails test:db                # Reset the database and run `bin/rails test`
+```
+
+### Database Setup
+
+The `db:create` and `db:drop` commands create or delete the database for the
+current environment (or all environments with the `db:create:all`,
+`db:drop:all`)
+
+The `db:seed` command loads sample data from `db/seeds.rb` and the
+`db:seed:replant` command truncates tables of each database for the current
+environment and then loads the seed data.
+
+The `db:setup` command creates all databases, loads all schemas, and initializes
+with the seed data (it does not drop databases first, like the `db:reset`
+command below).
+
+The `db:reset` command drops and recreates all databases from their schema for
+the current environment and loads the seed data (so it's a combination of the
+above commands).
+
+NOTE: For more on seed data, see [this
+section](active_record_migrations.html#migrations-and-seed-data) of the Active
+Record Migrations guide.
+
+### Migrations
+
+The `db:migrate` command is one of the most frequently run commands in a Rails
+application; it migrates the database by running all new (not yet run)
+migrations.
+
+The `db:migrate:up` command runs the "up" method and the `db:migrate:down`
+command runs the "down" method for the migration specified by the VERSION
+argument.
+
+```bash
+$ bin/rails db:migrate:down VERSION=20250812120000
 ```
 
-### `bin/rails assets:`
+The `db:rollback` command rolls the schema back to the previous version (or you
+can specify steps with the `STEP=n` argument).
+
+The `db:migrate:redo` command rolls back the database one migration and
+re-migrates up. It is a combination of the above two commands.
+
+There is also a `db:migrate:status` command, which shows which migrations have
+been run and which are still pending:
+
+```bash
+$ bin/rails db:migrate:status
+database: db/development.sqlite3
+
+ Status   Migration ID    Migration Name
+--------------------------------------------------
+   up     20250101010101  Create users
+   up     20250102020202  Add email to users
+  down    20250812120000  Add age to users
+```
+
+NOTE: Please see the [Migration Guide](active_record_migrations.html) for an
+explanation of concepts related to database migrations.
+
+### Schema Management
 
-You can precompile the assets in `app/assets` using `bin/rails assets:precompile`, and remove older compiled assets using `bin/rails assets:clean`. The `assets:clean` command allows for rolling deploys that may still be linking to an old asset while the new assets are being built.
+There are two main commands that help with managing the database schema in your
+Rails application: `db:schema:dump` and `db:schema:load`.
 
-If you want to clear `public/assets` completely, you can use `bin/rails assets:clobber`.
+**The `db:schema:dump` command** reads your database’s current schema and writes
```

Comment:
I can see emphasizing the command let's it stand out more, but we don't seem to do it for any of the other commands in this guide. So maybe not make it bold for now?
```suggestion
The `db:schema:dump` command reads your database’s current schema and writes
```